lib/ansible/galaxy/api.py

```diff
@@ -454,9 +454,16 @@ def wait_import_task(self, task_url, timeout=0):
         start = time.time()
         wait = 2
 
+        # v3 returns relative task urls, v3 returns full urls
```

Comment:
is one of these supposed to be v2?

Reply:
Yep. Should be `v3 returns relative task urls, v2 returns full urls`